src/base/bittorrent/torrenthandle.h

```diff
@@ -243,6 +243,9 @@ namespace BitTorrent
         virtual int downloadLimit() const = 0;
         virtual int uploadLimit() const = 0;
         virtual bool superSeeding() const = 0;
+        virtual bool disableDHT() const = 0;
```

Comment:
I would rename all these to `isXXXDisabled()`, `setXXXDisabled()`

Reply:
ok

Reply:
Why not simply ```disableXXX(bool disable)``` for the setters?

Reply:
I guess set... is easier to search for. Assuming all setters start with it (i didn't look).

Reply:
Yes, you're right.